### PR TITLE
760 - Bug fix: YouTube video in accordion appears outside of accordion

### PR DESCRIPTION
**Pull request by seanRong**
Note: I didn't have as large of an overflow issue without this fix, but I do believe the overflow has been reduced, this might need to be tested on the test site as well.

### CODE REVIEW

File: templates/vue/src/components/lightbox/AccordionMedia.vue [hunk 1]
```diff
@@ -184,7 +184,7 @@ button[disabled] {
 
 .media-container {
   height: 100%;
-  overflow: scroll;
+  overflow: hidden;
```
Comment (wynnset) [inline templates/vue/src/components/lightbox/AccordionMedia.vue:187]: This is not working for me. When my accordion is longer than the page, it doesn't let me scroll down to see the rest of the accordion.

**Review by wynnset — CHANGES_REQUESTED** [review]
See comment below. Additionally, my YouTube video is appearing too narrow: 
image

**Comment by yungalyx** [comment]
cannot test because I'm stuck in loop during video content/ upload youtube link to node. ):

**Comment by seanRong** [comment]
Tried it and it's working for me, what link did you use Alex?

Additionally, vue-youtube-embed module is giving red console errors like
```Error in callback for watcher "playerHeight": "TypeError: this.player.setSize is not a function"
```

But I'm not sure if we really want to update that module?

**Review by shirleywbi — APPROVED** [review]
Looks good.
image

**Review by wynnset — APPROVED** [review]

**Pull request merged**